etna/lib/etna/application.rb

```diff
@@ -76,6 +76,13 @@ def config(type, env = environment)
     @config[env][type]
   end
 
+  def env_config(env = environment)
```

Comment:
Used by the config show command  to attempt to get all of a single environment's config.